xsd/netex_part_3/part3_salesTransactions/netex_fareDebit_version.xsd

```diff
@@ -0,0 +1,351 @@
+		<xsd:complexType>
+			<xsd:complexContent>
+				<xsd:restriction base="FareDebit_VersionStructure">
+					<xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="DataManagedObjectGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="LogEntryGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="FareDebitGroup"/>
+						</xsd:sequence>
+					</xsd:sequence>
+					<xsd:attribute name="id" type="FareDebitIdType" use="optional">
+						<xsd:annotation>
+							<xsd:documentation>Identifier of ENTITY.</xsd:documentation>
+						</xsd:annotation>
+					</xsd:attribute>
+				</xsd:restriction>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	<xsd:complexType name="FareDebit_VersionStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for FARE DEBIT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="LogEntry_VersionStructure">
+				<xsd:sequence>
+					<xsd:group ref="FareDebitGroup"/>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:group name="FareDebitGroup">
+		<xsd:annotation>
+			<xsd:documentation>Elements for FARE DEBIT</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:group ref="PaymentAmountGroup"/>
+			<xsd:element name="CardNumber" type="PrivateCodeStructure" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Card number used,</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+		</xsd:sequence>
+	</xsd:group>
+	<!-- ==== BOOKING DEBIT ================================================ -->
+	<xsd:element name="BookingDebit" abstract="false" substitutionGroup="FareDebit_DummyType">
+		<xsd:annotation>
+			<xsd:documentation>A LOG ENTRY recording data about a debiting action for a booking fee.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexType>
+			<xsd:complexContent>
+				<xsd:restriction base="BookingDebit_VersionStructure">
+					<xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="DataManagedObjectGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="LogEntryGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="BookingDebitGroup"/>
+						</xsd:sequence>
+					</xsd:sequence>
+					<xsd:attribute name="id" type="BookingDebitIdType"/>
+				</xsd:restriction>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	<xsd:complexType name="BookingDebit_VersionStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for BOOKING DEBIT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="FareDebit_VersionStructure">
+				<xsd:sequence>
+					<xsd:group ref="BookingDebitGroup"/>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:group name="BookingDebitGroup">
+		<xsd:annotation>
+			<xsd:documentation>Elements for BOOKING DEBIT</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence/>
```

Comment:
What is this?

Reply:
From TM : _A LOG ENTRY recording data about a debiting action for a booking fee._ 
To be detailed in the doc

Reply:
I am talking about the empty sequence.